packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -51,52 +51,55 @@
       />
 
       <template #form-actions>
-        <!--
-          Force the render of this slot
-             - if isWizardStep is true we don't want any buttons displayed (default EntityBaseForm buttons included)
-        -->
-        <div v-if="isWizardStep" />
         <div
-          v-else
-          class="plugin-form-actions"
+          :id="`plugin-form-default-actions-container-${(config as KongManagerPluginFormConfig).workspace || (config as KonnectPluginFormConfig).controlPlaneId}`"
+          ref="actionsDivRef"
+        />
+
+        <!-- if isWizardStep is true we don't want any buttons displayed (default EntityBaseForm buttons included) -->
+        <Teleport
+          v-if="!isWizardStep && mounted"
```

Comment:
nit: Technically we should be using Teleport `disable` option to achieve this. The only downside is the warning which might come if we do not provide a portal destination id similar to the other issue we saw. Hopefully we can get Yi's PR merged in to get past that.